content/ko/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -14,49 +14,99 @@ card:
 이 페이지에서는 `kubeadm` 툴박스 설치 방법을 보여준다.
 이 설치 프로세스를 수행한 후 kubeadm으로 클러스터를 만드는 방법에 대한 자세한 내용은 [kubeadm으로 클러스터 생성하기](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/) 페이지를 참고한다.
 
+{{< doc-versions-list "installation guide" >}}
 
 ## {{% heading "prerequisites" %}}
 
+* 호환되는 리눅스 호스트. 쿠버네티스 프로젝트는 데비안 및 레드햇 기반 리눅스 배포판과
+   패키지 매니저가 없는 배포판에 대한 일반적인 지침을 제공한다.
+* 머신당 2GB 이상의 RAM (이보다 적으면 앱을 위한 공간이 거의 남지 않는다).
+* 컨트롤 플레인 머신에는 2개 이상의 CPU.
+* 클러스터의 모든 머신 간 완전한 네트워크 연결 (공용 또는 사설 네트워크 무관).
+* 모든 노드에 대해 고유한 호스트네임, MAC 주소, 그리고 product_uuid. 자세한 내용은 [여기](#verify-mac-address)를 참고한다.
+* 머신에서 특정 포트들이 열려있어야 한다. 자세한 내용은 [여기](#check-required-ports)를 참고한다.
 
-* 호환되는 리눅스 머신. 쿠버네티스 프로젝트는 데비안 기반 배포판, 레드햇 기반 배포판, 그리고 패키지 매니저를 사용하지 않는 경우에 대한 일반적인 가이드를 제공한다.
-* 2 GB 이상의 램을 장착한 머신. (이 보다 작으면 사용자의 앱을 위한 공간이 거의 남지 않음)
-* 2 이상의 CPU.
-* 클러스터의 모든 머신에 걸친 전체 네트워크 연결. (공용 또는 사설 네트워크면 괜찮음)
-* 모든 노드에 대해 고유한 호스트 이름, MAC 주소 및 product_uuid. 자세한 내용은 [여기](#verify-mac-address)를 참고한다.
-* 컴퓨터의 특정 포트들 개방. 자세한 내용은 [여기](#check-required-ports)를 참고한다.
-* 스왑의 비활성화. kubelet이 제대로 작동하게 하려면 **반드시** 스왑을 사용하지 않도록 설정한다.
+{{< note >}}
+`kubeadm` 설치는 동적 링킹을 사용하는 바이너리를 통해 수행되며 대상 시스템이 `glibc`를 제공한다고 가정한다.
+이는 많은 리눅스 배포판(데비안, 우분투, 페도라, CentOS 등 포함)에서 합리적인 가정이지만
```

Comment:
Fine with this - just note: I personally prefer to use distribution names as English words (e.g., Debian, Ubuntu, and Febora) rather than Korean but I see that some strings are already translated such ways (like this PR). Not sure whether it is already discussed or not via Korean L10n team.